### DISCUSSION TITLE
Add All In Folder

### DISCUSSION THREAD

**Discussion by teslasparks301** [rern/rAudio#493] [Feature Requests]
When browsing a folder in my library, which in my case is generally an album, I would like to be  able to add all the tracks in a single operation. If there is already a way to do this, please tell me how. So far, I have only been able to add tracks one at a time.

Thank you.
teslasparks301

**Comment by R101G** [comment]
Go back and click on the folder icon. You have the choice to add, replace, add+play or replace+play.

**Reply by teslasparks301** [reply]
Yay! It worked. It such a basic function for a music player I figured it was there someplace.  I kept clicking on the album name.
